leaf-ui/rappid-extensions/ConfigInspector.js

```diff
@@ -167,7 +170,18 @@ var Config = Backbone.View.extend({
      * and triggering a removal of its respective view 
      */
     removeConfig:function(){
-        this.model.destroy();
+        index = this.model.collection.indexOf(this.model);
```

Comment:
Thoughts on this logic?
if(model is selected && length of configCollection > 1)
     if(model index > 0) -> set model at index-1 to selected = true
     else -> set model at index + 1 to select = true
destroy model